src/plugins/noDevtoolsWarning/index.ts

```diff
@@ -21,7 +21,7 @@ import definePlugin from "@utils/types";
 
 export default definePlugin({
     name: "NoDevtoolsWarning",
-    description: "Disables the 'HOLD UP' banner in the console. As a side effect, also prevents Discord from hiding your token, which prevents random logouts.",
+    description: "Disables the 'HOLD UP' banner in the console. Also stops random logouts when using DevTools.",
```

Comment:
maybe something like this
```suggestion
    description: "Disables the 'HOLD UP' banner in the console. Also stops random logouts & token hiding when using DevTools.",
```

Reply:
I still think the original description is perfect as-is. No reason to be juggling words around.

Reply:
Actually.. this might be better:

"Disables the 'HOLD UP' banner in the DevTools console. As a side effect, this also prevents Discord from hiding your token and logging you out of your account when you close Discord while DevTools is open."